### DISCUSSION TITLE
New track elements have no spinning data

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#19923] [Feature requests]
Currently the 1/8th sloped turns (flat and banked) do not create a spinning moment on vehicles traversing. This should be fixed.

**Comment by X123M3-256** [comment]
It's not just the spinning data, there's a lot missing ... the prices, height markers, allowed wall edges, etc. I filled in dummy data when I started development because you have to have something there but I wanted to concentrate on getting the pieces functional, and then completely forgot to come back and fill it out properly. Seems none of the reviewers noticed either, so the PR was merged prematurely. I will have to submit a new PR to fix it ... when I get around to it.
